platform/darwin/scripts/generate-style-code.js

```diff
@@ -642,7 +642,7 @@ global.valueTransformerArguments = function (property) {
                 case 'anchor':
                     return ['std::vector<mbgl::style::SymbolAnchorType>', objCType, 'mbgl::style::SymbolAnchorType', 'MGLTextAnchor'];
                 case 'mode':
-                    return ['std::vector<mbgl::style::TextWritingModeType>', objCType, 'mbgl::style::TextWritingModeType', 'MGLTextWritingMode'];
+                    return ['std::vector<mbgl::style::TextWritingModeType>', objCType, 'mbgl::style::TextWritingModeType', 'MGLTextWritingModes'];
```

Comment:
Would it be possible to name the property `textWritingMode` but the type `MGLTextWritingMode`, singular? I think we’re running into this issue now because the style specification only recently introduced properties that are arrays of enumeration values.

Reply:
second this request